scss/main.scss

```diff
@@ -50,10 +50,10 @@ section {
   position: relative;
   #instructions {
     color: $greenFontColor;
-    padding: 1em 1em 3em 1em;
+    padding: 1em 1em;// 3em 1em;
     max-width: 368px;
     margin: 0 auto;
-    border-radius: 10px;
+    border-radius: 10px; // -- Why is this here?  - Cara
```

Comment:
Forgot to delete. It was initially there because I was experimenting with a colored background behind the instructions and gave that background rounded borders